### PR TITLE
chore: decide which env is going to be tested by the branch

### PR DESCRIPTION
**Pull request by amandakarina**
Decides which environments are going to be tested bases on the branch triggering the build.

### CODE REVIEW

File: test/integration/testutils/strings.go [hunk 1]
```diff
@@ -45,3 +41,16 @@ func GetLastSplitElement(value string, sep string) string {
 	splitted := strings.Split(value, sep)
 	return splitted[len(splitted)-1]
 }
+
+func EnvNames(branchName string) []string {
+	if branchName == "release-please--branches--main" {
```
Comment (apeabody) [inline test/integration/testutils/strings.go:46]: Is it possible to retrieve `utils.ValFromEnv(t, "TF_VAR_branch_name")` here? Reduce duplication in the code.

Comment (apeabody) [inline test/integration/testutils/strings.go:46]: I like making this a function, as longer term a switch/case can be used when we have different types of examples that use different env names.

**Review by apeabody — APPROVED** [review]
Thanks @amandakarina! One suggestion, but this LGTM.

**Review by caetano-colin — APPROVED** [review]

File: test/integration/testutils/strings.go (hunk 1 above)
Comment (amandakarina) [reply test/integration/testutils/strings.go:46]: Are you suggesting change the function to:

```suggestion
	func EnvNames(t *testing.T) []string {
	branchName := utils.ValFromEnv(t, "TF_VAR_branch_name")
	if branchName == "release-please--branches--main" {
		return []string{
			"development",
			"non-production",
			"production",
		}
	}
	return []string{
		"development",
	}
}
```

Comment (apeabody) [reply test/integration/testutils/strings.go:46]: Yeah, not a big deal, but that way only the single call to `EnvNames()` is needed in the various tests.  I'm also fine with the current design if there is a benefit to keep it flexible.

**Review by apeabody — APPROVED** [review]
Thanks @amandakarina!

**Pull request merged**